torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1070,6 +1070,10 @@ def sample_inputs_masked_var(op_info, device, dtype, requires_grad, **kwargs):
                     # autograd output that test_grad_fn cannot handle
                     # correctly.
                     continue
+                if sample_input.input.dim() == 0 or orig_count.min() == 1:
+                    # Skip samples when the autograd output for std could not be handled correctly
+                    # due to torch.sqrt
+                    continue
```

Comment:
This change is unnecessary if one corrects
https://github.com/pytorch/pytorch/blob/1f29b3130af218847a043e58fdc64511bbe072fe/torch/testing/_internal/common_methods_invocations.py#L1068
to
```python
        if orig_count.min() <= int(unbiased) + 1:
```
```suggestion
```